src/components/forms/auto-complete/RuiAutoComplete.vue

```diff
@@ -447,6 +447,16 @@ function getListeners(on: Record<string, Function | undefined>, $listeners: Reco
     ...listenersFiltered,
   };
 }
+
+function setInputSelectionRange(start: number, end: number) {
+  set(searchInputFocused, true);
+  get(textInput)?.setSelectionRange?.(start, end);
+}
+
+defineExpose({
+  setInputFocus,
+  setInputSelectionRange,
```

Comment:
Would it make sense to expose them as `focus`/`setSelectionRange` externally?

Reply:
updated